tests/object/omit.test.ts

```diff
@@ -21,10 +21,17 @@ describe('omit', () => {
     expect(result).toEqual(person)
   })
   test('returns object without omitted properties', () => {
-    const result = _.omit(person, ['name'])
+    const result = _.omit(person, ['name'] )
     expect(result).toEqual({
       age: 20,
       active: true,
     })
   })
+  test('returns object without omitted properties when keys are readonly', () => {
```

Comment:
We don't need a runtime test here, since this PR only affects type-checking.

We *could* add a ”type test”, but this change is simple enough that I think that too would be overkill.

Reply:
@aleclarson If it's an overhead then I think it's better to remove this test. It's a simple change.

Reply:
I will remove it.